plugins/@grouparoo/iterable/src/lib/export/exportProfile.ts

```diff
@@ -0,0 +1,98 @@
+import { ExportProfilePluginMethod } from "@grouparoo/core";
+import { connect } from "../connect";
+import { addToList, removeFromList } from "./listMethods";
+
+export const exportProfile: ExportProfilePluginMethod = async ({
+  appGuid,
+  appOptions,
+  export: {
+    toDelete,
+    newProfileProperties,
+    oldProfileProperties,
+    newGroups,
+    oldGroups,
+  },
+}) => {
+  // if we received no mapped data... just exit
+  if (Object.keys(newProfileProperties).length === 0) {
+    return { success: true };
+  }
+  const client = await connect(appOptions);
+
+  const email = newProfileProperties["email"]; // this is how we will identify profiles
+  const currentEmail = oldProfileProperties["email"];
+  if (!email) {
+    throw new Error(`newProfileProperties[email] is a required mapping`);
+  }
+
+  // Remove email from profile properties to avoid duplicates.
+  delete oldProfileProperties.email;
+  delete newProfileProperties.email;
+
+  try {
+    if (toDelete) {
+      await client.users.delete(email);
+    } else {
+      // create the user and set properties
+      const deletePropertiesPayload = {};
+      const newPropertyKeys = Object.keys(newProfileProperties);
+      Object.keys(oldProfileProperties)
+        .filter((k) => !newPropertyKeys.includes(k))
+        .forEach((k) => (deletePropertiesPayload[k] = null));
+
+      const dataFields = Object.assign(
+        {},
+        newProfileProperties,
```

Comment:
it's a minor thing, but I would suggest the other other order to make sure that `newProperfileProperties` always "wins"
```
const dataFields = Object.assign(
        {},
        deletePropertiesPayload
        newProfileProperties
)
```